app/services/audit/AuditService.scala

```diff
@@ -149,15 +133,12 @@ class AuditService @Inject() (connector: AuditConnector, appConfig: AppConfig)(i
         logger.warn(s"Cannot strip empty fields from Json:\n\t$msg")
         json
     }
-
-  private def tagsForMessageInboxPartialRetrieved(path: String)(implicit hc: HeaderCarrier): Map[String, String] =
-    AuditExtensions.auditHeaderCarrier(hc).toAuditTags(transactionName = "callExportPartial", path = path)
 }
 
 object AuditTypes extends Enumeration {
   type Audit = Value
-  val SaveDraftValue, Submission, SubmissionPayload, Cancellation, CancellationPayload, NavigateToMessages, Amendment, AmendmentPayload,
```

Comment:
Are we sure 'NavigateToMessages' isn't used for the SFUS links? :thinking:

Reply:
It was only used when auditing an event for partial retrieved messages to the inbox, so it was expected to be removed